src/main/proto/netflix/titus/agent.proto

```diff
@@ -292,6 +292,9 @@ message ContainerInfo {
 
     /// (Optional) IP address allocation for the task
     optional com.netflix.titus.SignedAddressAllocation signedAddressAllocation = 41;
+
+    /// Timestamp in epoch millis of when the task's job was Accepted
+    optional uint64 jobAcceptedTimestamp = 42;
```

Comment:
Might be worth indicating that it's epoch milliseconds in the name, similar to what we do here: https://github.com/Netflix/titus-api-definitions/blob/master/src/main/proto/netflix/titus/agent.proto#L312